### DISCUSSION TITLE
Hide / Restrict access to cube

### DISCUSSION THREAD

**Discussion by EricDallAgnol** [atoti/atoti#790] [Q&A] [answered]
Hi team 👋 

In our project we have several cubes and we are wondering if there are some ways to hide from the discovery call or to restrict the usage of some of these cubes to users ?

Many thanks !
Eric

**Comment by tibdex** [comment, marked as answer]
Hi Eric, at the moment the cube visibility cannot be changed and no restrictions can be applied to users to hide cubes from them. You could open a Jira ticket to describe your use case in more details.

Thanks
